### PR TITLE
union of unsorted array

### PR DESCRIPTION
**Pull request by Sapna2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)
Yes

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#3771

### CODE REVIEW

**Comment by Sapna2001** [comment]
Replit : https://replit.com/@Sapna2001/unionUnsorted

File: Java/other/UnionUnsortedArray.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* Program to find the union of two unsorted array */
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class UnionUnsortedArray {
+
+	public static void main(String[] args) {
```
Comment (HarshCasper) [inline Java/other/UnionUnsortedArray.java]: Don't put all the code logic in the `main()` function.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Use `main()` only for the driver code

**Comment by Sapna2001** [comment]
@HarshCasper I have made changes in the main method

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**